ansys/mapdl/core/license.py

```diff
@@ -0,0 +1,349 @@
+    # subprocess.check_output(command, shell=os.name != 'nt')
+    process =  subprocess.Popen(command,
+                           stdout=subprocess.PIPE,
+                           stderr=subprocess.STDOUT)
+    return process.stdout.read().decode()
+
+
+def get_lmutil_path():
+    if os.name == 'nt': # Windows
+        lmutil_path = get_lmutil_path_windows()
+
+    elif os.name == 'posix':  # Linux
+        lmutil_path = get_lmutil_path_linux() 
+    
+    return os.path.join(lmutil_path, 'lmutil.exe')
+
+
+def get_lmutil_path_windows():
+    ansyslic_dir = os.getenv('ANSYSLIC_DIR')
+    return os.path.join(ansyslic_dir, 'winx64')
+
+
+def get_lmutil_path_linux():
+    ansyslic_dir = os.getenv('ANSYSLIC_DIR')
+    return os.path.join(ansyslic_dir, 'linx64') #TODO: Check if this folder is right.
+
+
+## ansysli_util
+
+def try_ansysli_util():
+    try:
+        check_license_server_with_ansysli_util()
+    except LicenseServerConnectionError:
+        # Reraising error 
+        raise 
+    except:
+        # Unkw
+        return False 
+    else:
+        return True 
+
+
+def check_license_server_with_ansysli_util():
+    """
+    Check the license server status by running 'ansysli_util'. 
+    """
+    warnings.warn('This method to check the license server status is not completely error free.\nIt is very likely it will show license not available.')
+    licenses = ['meba']  # mechanical enterprise license. 
+
+    for each_license in licenses:
+        output = run_ansysli_util(each_license)
+        msg1 = 'No such feature exists'
+        msg2 = 'The server is down or is not responsive.'
+        if msg1 in output or msg2 in output:
+            raise LicenseServerConnectionError(head_message=f"'ansysli_util' reports '{msg1}' or '{msg2}'")
+
+
+def run_ansysli_util(license):
+    ansysli_util_path = get_ansysli_util_path()
+    command = f"{ansysli_util_path}  -checkout {license} "
+
+    # subprocess.check_output(command, shell=os.name != 'nt')
+    process =  subprocess.Popen(command,
+                           stdout=subprocess.PIPE,
+                           stderr=subprocess.STDOUT)
+    return process.stdout.read().decode()
+
+
+def get_ansysli_util_path():
+    if os.name == 'nt': # Windows
+        ansysli_util_path = get_ansysli_util_path_windows()
+
+    elif os.name == 'posix':  # Linux
+        ansysli_util_path = get_ansysli_util_path_linux() 
+    
+    return os.path.join(ansysli_util_path, 'ansysli_util.exe')
+
+
+def get_ansysli_util_path_windows():
+    ansyslic_dir = os.getenv('ANSYSLIC_DIR')
+    return os.path.join(ansyslic_dir, 'winx64')
+
+
+def get_ansysli_util_path_linux():
+    ansyslic_dir = os.getenv('ANSYSLIC_DIR')
+    return os.path.join(ansyslic_dir, 'linx64') #TODO: Check if this folder is right.
+
+
+## Main 
+
+def try_license_server():
+    """
+    Trying the three possible methods to check the license server status.
+    """
+    success_ = try_license_file()
```

Comment:
```suggestion
    status = try_license_file()
```
I don't think the trailing underscore is necessary here. The variable name can just be `success` or `status`.